src/vellum/workflows/events/node.py

```diff
@@ -21,6 +22,15 @@ class _BaseNodeExecutionBody(UniversalBaseModel):
     def serialize_node_definition(self, node_definition: Type, _info: Any) -> Dict[str, Any]:
         return serialize_type_encoder(node_definition)
 
+    # Couldn't get this to work with model_config.exclude_none or model_config.exclude_defaults
+    # so we're excluding null invoked_ports manually here for now
```

Comment:
p99 of streaming events won't have invoked ports emitted, so we want to exclude them from the dump